reference/3.0/Microsoft.PowerShell.Utility/Out-File.md

```diff
@@ -1,108 +1,141 @@
 ﻿---
-ms.date:  11/09/2017
+ms.date:  2/6/2019
 schema:  2.0.0
 locale:  en-us
 keywords:  powershell,cmdlet
 online version:  http://go.microsoft.com/fwlink/?LinkID=113363
 external help file:  Microsoft.PowerShell.Commands.Utility.dll-Help.xml
 title:  Out-File
 ---
+
 # Out-File
 
 ## SYNOPSIS
-
 Sends output to a file.
 
 ## SYNTAX
 
 ### ByPath (Default)
 
-```powershell
-Out-File [-FilePath] <String> [[-Encoding] <String>] [-Append] [-Force] [-NoClobber] [-Width <Int32>]
- [-InputObject <PSObject>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+Out-File [-FilePath] <string> [[-Encoding] <string>] [-Append] [-Force] [-NoClobber] [-Width <int>]
+[-InputObject <psobject>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### ByLiteralPath
 
-```powershell
-Out-File -LiteralPath <String> [[-Encoding] <String>] [-Append] [-Force] [-NoClobber] [-Width <Int32>]
- [-InputObject <PSObject>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+Out-File [[-Encoding] <string>] -LiteralPath <string> [-Append] [-Force] [-NoClobber] [-Width <int>]
+[-InputObject <psobject>] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
 
-The `Out-File` cmdlet sends output to a file.
-You can use this cmdlet instead of the redirection operator (\>) when you need to use its parameters.
+The `Out-File` cmdlet sends output to a file. When you need to specify parameters for the output use
+`Out-File` rather than the redirection operator (`\>`).
```

Comment:
You don't need the backslash when you put it a code fence. So the redirection operator should be (\`>\`). Fix this in all versions.

Reply:
Thanks. I made the change in all versions and updated the PR.